src/test/java/org/magicdgs/readtools/utils/trimming/TrimAndFilterPipelineUnitTest.java

```diff
@@ -429,7 +431,7 @@ private void testTrimmingMetric(final TrimmerMetric metric,
     public void testGetEmptyPipelineBlowsUp() throws Exception {
         // empty argument collection
         TrimAndFilterPipeline.fromPluginDescriptors(
-                new TrimmerPluginDescriptor(null),
+                new TrimmerPluginDescriptor(new TrimReadsTrimmerPluginArgumentCollection(), null),
```

Comment:
This can use easily an simpler argument collection without arguments to blow up.

Reply:
Done.